package.json

```diff
@@ -26,62 +26,59 @@
   },
   "dependencies": {
     "@hmcts/cookie-manager": "1.0.0",
-    "@hmcts/info-provider": "^1.2.2",
+    "@hmcts/info-provider": "^1.2.3",
     "@hmcts/nodejs-healthcheck": "^1.8.5",
     "@hmcts/nodejs-logging": "^4.0.4",
     "@hmcts/properties-volume": "^1.2.0",
+    "@npmcli/fs": "^4.0.0",
     "@types/config": "^3.3.5",
     "@types/cookie-parser": "^1.4.8",
+    "@types/csurf": "^1.11.5",
     "@types/es6-promisify": "^6.0.4",
     "@types/express": "^5.0.0",
     "@types/glob": "^8.1.0",
     "@types/helmet": "^0.0.48",
-    "@types/http-proxy-middleware": "^1.0.0",
-    "@types/lodash": "^4.17.13",
-    "@types/node": "^18.19.70",
+    "@types/lodash": "^4.17.16",
+    "@types/node": "^18.19.80",
     "@types/nunjucks": "^3.2.6",
     "@types/require-directory": "^2.1.6",
     "@types/serve-favicon": "^2.5.7",
     "applicationinsights": "^2.9.6",
     "autobind-decorator": "^2.4.0",
     "awilix": "^7.0.3",
-    "axios": "1.7.9",
+    "axios": "1.8.3",
     "cacache": "^19.0.1",
     "config": "^3.3.12",
     "cookie-parser": "^1.4.7",
-    "csrf-csrf": "^3.1.0",
+    "csurf": "^1.11.0",
```

Comment:
how come you brought back csurf? it's deprecated i think?

Reply:
don't think i was meant to change that my bad, changed back